smac/tae/execute_func.py

```diff
@@ -94,18 +100,18 @@ def run(self, config, instance=None,
 
         Parameters
         ----------
-            config : dictionary (or similar)
+            config : Configuration, dictionary (or similar)
                 Dictionary param -> value
-            instance : str
+            instance : str, optional
                 Problem instance
-            cutoff : int, optional
+            cutoff : float, optional
```

Comment:
Are you sure about this? The pynisher can't handle float cutoffs.

Reply:
The signature in `execute_func` (int) and `execute_ta_run` (float) were different. I thought this will make it more consistent. Since we convert cutoff to int before using in pynisher anyway, I thought this wouldn't affect it.

Reply:
Since there's a cast, how about you leave these as `float` and update the docstring.